### PR TITLE
Add logic for packing CLI into executables for platforms

### PR DESCRIPTION
**Pull request by sigalsax**
### What does this PR do?
- _What's changed? Why were these changes made?_
- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### What ticket does this PR close?
Resolves #[relevant GitHub issues, eg 76]

#### Change log
- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [ ] The changes in this PR do not require tests

#### Documentation
- [ ] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [ ] This PR does not require updating any documentation

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -13,93 +13,18 @@ pipeline {
...
-      when {
-        branch "master"
-      }
-    }
-  }
-
-  post {
-    always {
-      cleanupAndNotify(currentBuild.currentResult)
-    }
```
Comment (sgnn7) [inline Jenkinsfile:103]: This should not get merged as-is. We lose testing, scanning, and publishing if this gets committed in its current state.

File: bin/pack_executable_windows [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+docker run --rm -it \
+  -v $(pwd)/dist:/opt/conjur-api-python3/dist \
+  conjur-api-python3-windows \
+    $BIN_BUILD_CMD
+
+echo $ls
+
+echo "The package"
+echo "$(cat $(pwd)/dist)"
```
Comment (sgnn7) [inline bin/pack_executable_windows:28]: You may be able to reuse the main building script as long as the image is different.

File: Jenkinsfile (hunk 1 above)
Comment (sigalsax) [reply Jenkinsfile:103]: totally. This is a draft PR that I used to experiment

**Comment by sigalsax** [comment]
Closing PR because automation will not be added during this stage

**Pull request closed without merging**